frontend/src/concepts/modelRegistry/utils/updateTimestamps.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  ModelRegistryAPIs,
+  ModelState,
+  ModelRegistryMetadataType,
+} from '~/concepts/modelRegistry/types';
+
+type MinimalModelRegistryAPI = Pick<ModelRegistryAPIs, 'patchRegisteredModel'>;
+
+export const bumpModelVersionTimestamp = async (
+  api: ModelRegistryAPIs,
+  modelVersionId: string,
+): Promise<void> => {
+  if (!modelVersionId) {
+    throw new Error('Model version ID is required');
+  }
+
+  try {
+    await api.patchModelVersion({}, { state: ModelState.LIVE }, modelVersionId);
+  } catch (error) {
+    throw new Error(
+      `Failed to update model version timestamp: ${
+        error instanceof Error ? error.message : String(error)
+      }`,
+    );
+  }
+};
+
+export const bumpRegisteredModelTimestamp = async (
+  api: MinimalModelRegistryAPI,
+  registeredModelId: string,
+): Promise<void> => {
+  if (!registeredModelId) {
+    throw new Error('Registered model ID is required');
+  }
+
+  try {
+    const currentTime = new Date().toISOString();
+    await api.patchRegisteredModel(
+      {},
+      {
+        state: ModelState.LIVE,
+        customProperties: {
+          _lastModified: {
+            metadataType: ModelRegistryMetadataType.STRING,
+            // eslint-disable-next-line camelcase
+            string_value: currentTime,
+          },
+        },
```

Comment:
Thinking more - if the `_lastModified` addition here was because the { state } alone wasn't enough to automatically bump the version, that's an issue we should raise again with the backend team. If we get blocked on this and it looks like it won't make 2.17 we could remove it from the strat scope.